lua/entities/cfc_shaped_charge/init.lua

```diff
@@ -2,6 +2,11 @@ AddCSLuaFile( "cl_init.lua" )
 AddCSLuaFile( "shared.lua" )
 include( "shared.lua" )
 
+local breakableClasses = {
+    ["prop_physics"] = true,
+    ["sent_spawnpoint"] = true
```

Comment:
I think all classes will be valid table keys too
```suggestion
    prop_physics = true,
    sent_spawnpoint = true
```